src/core/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -8,6 +8,7 @@ import { render } from '@testing-library/react';
 import { Breadcrumbs, BreadcrumbsProps } from './Breadcrumbs';
 import { Button } from '../Buttons';
 import { SvgChevronRight } from '@itwin/itwinui-icons-react';
+import * as UseOverflow from '../utils/hooks/useOverflow';
```

Comment:
Why need to import all?

Reply:
Because for `spyOn` you need object and its property name. Or you know a better way to mock a function?

Reply:
I see there is way if you have default export.. you can just pass path to spy, but.. Maybe this is giving better readability